src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java

```diff
@@ -59,6 +60,9 @@
 @Profile("hardware")
 public class LockKeeperServiceImpl implements LockKeeperService {
 
+    public static final String SWITCH_TYPE_PARAMETER = "switch_type";
+    public static final String NOVIFLOW_SWITCH_TYPE = "noviflow";
```

Comment:
As long as you don't use these constants separately, it's better to join them into one, e.g.
```
private static final HashMap<String, String> NOFIFLOW_SWITCH_PARAMETERS = ImmutableMap.of("switch_type",  "noviflow") 
```
(For some reason we have guava in test-library dependencies).